lib/elixir/test/elixir/test_helper.exs

```diff
@@ -13,6 +13,13 @@ defmodule PathHelpers do
     Path.expand("fixtures", __DIR__)
   end
 
+  def tmp_fixture_path(extra) do
```

Comment:
Because this is used only by `Rename`, let's keep it in the Rename module for now.